sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/DicomIO.java

```diff
@@ -51,8 +126,18 @@
  * readMetadataResult.getReadResponse() PCollection<String> failRead =
  * readMetadataResult.getFailedReads() }
  */
+
+ @SuppressWarnings({
+  "nullness" // TODO(https://github.com/apache/beam/issues/20497)
```

Comment:
Can we remove this from the class? There are ways to achieve this by using methods in [org.apache.beam.sdk.util.Preconditions](https://github.com/apache/beam/blob/master/sdks/java/core/src/main/java/org/apache/beam/sdk/util/Preconditions.java)